easycv/hooks/optimizer_hook.py

```diff
@@ -103,11 +104,17 @@ def __init__(self,
         self.ignore_key_epoch = ignore_key_epoch
         if LooseVersion(torch.__version__) >= LooseVersion('1.6.0'):
             self.scaler = amp.GradScaler()
+            self._scale_update_param = 512.
 
     def before_run(self, runner):
-        runner.fp16_enable = True
+        """Preparing steps before Mixed Precision Training."""
+        # wrap model mode to fp16
+        wrap_fp16_model(runner.model)
```

Comment:
this is `AMPFP16OptimizerHook`, support mixed precision

but `wrap_fp16_model` use model.half() , convert all the model weights to half precision.

remove `wrap_fp16_model`

Reply:
> 

uniform use mmcv fp16